src/opentype/woff2.js

```diff
@@ -2,6 +2,13 @@ import { SimpleTable } from "./tables/simple-table.js";
 import lazy from "../lazy.js";
 
 const brotliDecode = globalThis.unbrotli;
+let nativeBrotliDecode = undefined;
+
+if (!brotliDecode) {
+  import("zlib").then((zlib) => {
+    nativeBrotliDecode = (buffer) => zlib.brotliDecompressSync(buffer);
+  });
+}
```

Comment:
This is a dynamic import, but for a native module, so while in theory this might kick in "after everything else", in reality it gets executed immediately.